docs/sources/administration/plugin-management/index.md

```diff
@@ -81,6 +83,11 @@ The Plugin catalog allows you to browse and manage plugins from within Grafana.
 _Video shows the Plugin catalog in a previous version of Grafana._
 
 In order to be able to install / uninstall / update plugins using plugin catalog, you must enable it via the `plugin_admin_enabled` flag in the [configuration]({{< relref "../../setup-grafana/configure-grafana/#plugin_admin_enabled" >}}) file.
+
+{{% admonition type="note" %}}
+In the latest versions of Grafana, the `plugin_admin_enabled` flag is enabled by default.
```

Comment:
I'd be tempted to just remove this and the previous paragraph instead.

Reply:
Acknowledge